src/applications/resources-and-support/hooks/useGetSearchResults.js

```diff
@@ -23,8 +21,141 @@ const countKeywordsFound = (searchableString = '', keyword = '') => {
   return keywordsFound;
 };
 
-const countWholePhraseMatchesFound = (searchableString = '', keyword = '') =>
-  searchableString.toLowerCase()?.split(keyword.toLowerCase())?.length - 1;
+export const countWholePhraseMatchesFound = (
```

Comment:
Looks like a lot of changes here, but I just made a bunch of utility functions out of the code that was in the `useEffect` so I could access it for unit testing.